src/Electrical/Analog/sources.jl

```diff
@@ -1,744 +1,33 @@
-# Define and register smooth functions
-_cos_wave(t, f, A, st, ϕ) = A*cos(2*π*f*(t - st) + ϕ)
-_sin_wave(t, f, A, st, ϕ) = A*sin(2*π*f*(t - st) + ϕ)
-_damped_sine_wave(t, f, A, st, ϕ, d) = exp((st-t)*d)*A*sin(2*π*f*(t-st) + ϕ)
-_ramp(t, δ, st, et, h) = h/(et-st)*(_xH(t, δ, st) - _xH(t, δ, et))
-_square_wave(t, δ, f, A, st) = A*2atan(sin(2π*(t-st)*f)/δ)/π
-_step(t, δ, h, a) = h*(atan((t-a)/δ)/π + 0.5)
-_triangular_wave(t, δ, f, A, st) = A*(1-2acos((1 - δ)sin(2π*(t-st)*f))/π)
-_xH(t, δ, tₒ) = (t-tₒ)*(1+((t-tₒ)/sqrt((t-tₒ)^2+δ^2)))/2
```

Comment:
Don't throw these away.

Reply:
I want to add them to Blocks. (But with a separate PR)